src/browser/api/notifications/shapes.ts

```diff
@@ -31,6 +31,10 @@ export interface NotificationMessage {
     data: any;
 }
 
+export interface NoteId extends Identity {
```

Comment:
What's with the name of this interface? Seemingly the shape of this declaration is missing any kind of identity information at all...